src/backend/src/services/projects.services.ts

```diff
@@ -749,15 +749,9 @@ export default class ProjectsService {
       throw new NotFoundException('Manufacturer', name);
     }
 
-    if (manufacturer.dateDeleted) throw new DeletedException('Manufacturer', manufacturer.name);
-
-    const dateDeleted: Date = new Date();
-    const deletedManufacturer = await prisma.manufacturer.update({
+    const deletedManufacturer = await prisma.manufacturer.delete({
       where: {
         name: manufacturer.name
-      },
-      data: {
-        dateDeleted
       }
     });
```

Comment:
can you wrap the deletedManufacturer being returned in the manufacturerTransformer